README.md

```diff
@@ -34,8 +34,25 @@ cd ../examples
 yarn && yarn run start
 ```
 
-If you now navigate to [localhost:8080](https://localhost:8080), you will see a list of examples
-that can be explored in the browser.
+If you now navigate to [localhost:3000](https://localhost:3000), the browser should show a simple 3D model with a UI.
+
+To show a model from your own CDF project, you must first configure Reveal to use your CDF credentials. Follows the steps in the section [Environment for Credentials](environment-for-credentials).
+
+change the URL to `https://localhost:3000/?project=<project>&env=<environment>&modelId=<modelId>&revisionId=<revisionId>`.
+
+Here, `<project>` is the name of the CDF project you want to connect to, `<environment>` is the name of the environment specified in the `.env` file (e.g. `example_environment`), and `<modelId>` and `<revisionId>` are the ids of the model you want to visualize. You can find these IDs in Cognite Fusion, in the "Upload 3D models" / "Manage 3D" subapp.
+
+### Troubleshooting
+
+Sometimes, especially the first time you log in to an environment, you may get a 401 error in the browser, which, after clicking through, resets the browser window to the default model view. In this case, there should be a `#code=...`-segment appended to the URL in the browser. Copy the URL you tried to visit (including the project, environment and IDs), and paste it to the left of the `#`-sign in the URL, replacing everything that was there before. In other words, your URL should now look like
+
+```
+https://localhost:3000/?project=<project>&env=<environment>&modelId=<modelId>&revisionId=<revisionId>#code=<access token>
```

Comment:
Should we consider to add #code=.... after the access token as that is how the new URL will look like?

Reply:
I did add the `#code=` there? What do you mean?

Reply:
My bad, I see that #code= now